### PR TITLE
Add summoner Boss class

### PR DESCRIPTION
**Pull request by c86ec23b-fef1-4979-b2fa-b9adc351b8cc**
### Why:

Closes [issue link]

### Confirm the following:
- [x] I have tested these changes (by compiling, running, and playing) and have seen no differences in gameplay
@ABCxFF

### CODE REVIEW

File: src/Entity/Tank/Projectile/Bullet.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export default class Bullet extends LivingEntity {
         this.relations.values.team = barrel.relations.values.team;
         this.relations.values.owner = tank;
 
-        this.physics.values.sides = 1;
+        this.physics.values.sides = bulletDefinition.sides || 1;
```
Comment (ABCxFF) [inline src/Entity/Tank/Projectile/Bullet.ts:76]: Maybe use `??` instead of `||` for the bulletDefinition.sides, that way people can set it to 0 (rare).

**Review by ABCxFF — APPROVED** [review]
LGTM besides that - up to you if you want to fix.

**Review by ABCxFF — CHANGES_REQUESTED** [review]
Forgot one thing - Should add Summoner into the spawneable boss list for the Arena

**Comment by c86ec23b-fef1-4979-b2fa-b9adc351b8cc** [comment]
@ABCxFF it fixed, looks like the file didnt save

**Review by ABCxFF — APPROVED** [review]

**Pull request merged**